libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -309,6 +309,16 @@ bool StaticAnalyzer::visit(FunctionCall const& _functionCall)
 							"Arithmetic modulo zero."
 						);
 		}
+
+		if (m_currentContract->isLibrary())
+			if (functionType->kind() == FunctionType::Kind::DelegateCall)
+				for (auto const& contractFunction: m_currentContract->definedFunctions())
+					if (functionType->declaration().id() == contractFunction->id())
```

Comment:
What about using `funcitonType->declaration()->scope()` instead of the for loop?

Reply:
Wouldn't we then loose the location of the function declaration, which is currently used for the secondary source location?

Reply:
`functionTyu->declaration()->sourceLocation`?

Reply:
Ah, ok. Got it and changed accordingly.